resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java

```diff
@@ -111,34 +121,74 @@ public Tuple2<Map<String, String>, ResidentTransactionEntity> getCommonTemplateV
         }
         return Tuples.of(templateVariables, residentTransactionEntity);
     }
+    private String getAttributeValue(String attributes, String languageCode) {
```

Comment:
Please add comment with description of what this method does.

Reply:
done.